javascript_nodejs/6.using-cards/bot.js

```diff
@@ -0,0 +1,287 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActivityTypes, CardFactory } = require('botbuilder');
+const { ChoicePrompt, DialogSet, DialogTurnResult, DialogTurnStatus, ListStyle } = require('botbuilder-dialogs');
+
+/**
+ * RichCardsBot prompts a user to select a Rich Card and then returns the card that matches the user's selection.
+ */
+class RichCardsBot {
+    
+    /**
+     * Constructs the three pieces necessary by this bot to operate:
+     * 1. StatePropertyAccessor
+     * 2. DialogSet
+     * 3. ChoicePrompt
+     * 
+     * The only argument taken by this constructor is a ConversationState instance, although any BotState instance would suffice for this bot. `conversationState` is used to create a StatePropertyAccessor, which is needed to create a DialogSet. All botbuilder-dialogs `Prompts` need a DialogSet to operate.
```

Comment:
> `conversationState` [](start = 142, length = 20)

This comment doesn't fit on my 4k monitor (!!). Please break it up into multiple lines.